File: SVProgressHUD使用/SVProgressHUD使用/ViewController.m
//
//  ViewController.m
//  SVProgressHUD使用
//
//  Created by zl on 2019/6/13.
//  Copyright © 2019 ZHUNJIEE. All rights reserved.
//

#import "ViewController.h"
#import "SVProgressHUD+HBW.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
}

- (IBAction)buttonDidClick:(UIButton *)sender {
    [SVProgressHUD showText:@"正在加载中..."];
}

@end

File: CircleProgressBar/CircleProgressBar/ViewController.swift
//
//  ViewController.swift
//  CircleProgressBar
//
//  Created by Houge on 2020/5/4.
//  Copyright © 2020 QTTX. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    @IBOutlet weak var circleV: CircleView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }


    @IBAction func animateStoryboard(_ sender: UIButton) {
        circleV.animateCircle(duration: 12.0)
    }
}

